cmd/utils/flags.go

```diff
@@ -421,6 +421,16 @@ var (
 		Name:  "miner.extradata",
 		Usage: "Block extra data set by the miner (default = client version)",
 	}
+	CeloFeeCurrencyDefault = cli.Float64Flag{
+		Name:  "celo.feecurrency.default",
+		Usage: "Default fraction of block gas limit available for TXs paid with a whitelisted currency",
+		Value: 0.9,
```

Comment:
This is the default value for any new gas token, the idea was to have at least some block space allocated to CELO. If we set it to one and validators don't bother to change the value, we might end up in a situation we tried to avoid.